### DISCUSSION TITLE
Misorientation at grain boundaries

### DISCUSSION THREAD

**Discussion by Debo-rah** [mtex-toolbox/mtex#1022] [Ask Anything] [answered]
Hi all,

we want to evaluate the misorientation at the grain boundaries. We only want the misorientation directly at the boundary, not the average misorientation between the grains. There where several solutions in the documentation and we were wondering whether we took the correct one? Can anyone help us?
```

[grains,ebsd.grainId,ebsd.mis2mean] = calcGrains(ebsd('C'),'threshold',[6*degree, 15*degree],'boundary','tight');
gB=grains.boundary('C','C');

figure
plot(gB,gB.misorientation.angle./degree,'linewidth',1.5)

figure
plotAngleDistribution(gB.misorientation)
saveFigure([FigName 'Hist.png']);
```

Thanks a lot!

**Comment by zmichels** [comment, marked as answer]
That approach looks to be what you want. The misorientations you have targeted are those between orientations on either side of each boundary segment.

Cheers,
Zach

**Comment by Debo-rah** [comment]
Perfect.
Thanks a lot for the fast reply.
